src/features/MovieList/getDataFromApi.js

```diff
@@ -1,5 +1,7 @@
-export const getDataFromApi = async (api) => {
-  const response = await fetch(api);
+export const getDataFromApi = async (page) => {
+  const response = await fetch(
+    `https://api.themoviedb.org/3/movie/popular?page=${page}&api_key=5808b0503fd4aaf8a5636df1649fe0dc`
+  );
 
   if (!response.ok) {
     new Error(response.statusText);
```

Comment:
It would be better if you change it to:
```jsx
if (!response.ok) {
   throw new Error(response.statusText);
```
this will help us avoid potential errors!